delivery_carrier_seur/README.rst

```diff
@@ -0,0 +1,39 @@
+Module for SEUR carrier labels
+==============================
+
+This module allows to:
+ * Generate label and tracking number
+ * Generate manifesto
+
+TODO
+----
+
+ * Cash on delivery
+ * Edit/delete delivery tracking
+ * Customs management
+ * ...
+
+EXTRA DEPENDENCIES
+------------------
+This module use the Seur API: https://pypi.python.org/pypi/seur
```

Comment:
Add also a usage and configuration section explaining how to use and configure it